encrypter.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace std;
+
+string read_file(string& address, string error_message, bool clean = false)
```

Comment:
File is a stream. Any stream can be iterated. There is a standard function which copies from one iterator to another.
This is from a search in google for "ifstream ofstream std::copy"
https://submit.ncbi.nlm.nih.gov/subs/wgs/SUB670025/general_info

Look for "COPY-ALGORITHM-C++-WAY" it's kind of an official way for C++ now.

But in your particular case it looks better just make << of source file rdbuf to ostringstream.
iIt's similar to KISS-C++-Streambuffer-WAY but read about std::ostringstream.